code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -271,3 +271,11 @@
 		user.emote("scream")
 		user.drop_all_held_items()
 		user.Paralyze(80)
+
+//SKYRAT ADDITION START
+/obj/item/gun/ballistic/revolver/cosmonaut
+	name = "Cosmonaut's Revolver"
+	desc = "You know what? Fuck you. *bullpups your revolver*"
+	icon_state = "cosmonaut"
```

Comment:
```suggestion
  icon = 'modular_skyrat/modules/modular_weapons/icons/obj/guns/projectile.dmi'
	icon_state = "cosmonaut"
```

Reply:
fucking indentation is hard in the web editor but you get the point

Reply:
I actually already tried this once, under recommendation from someone on the Discord, but doing it that way made it far harder to locate, made less sense, and just looked like shitcode. Doing it in the normal areas makes it simpler to find, and make more sense.

Reply:
I would happily do this, but I can't find **ANY** of the locations you're referring to.

Reply:
Plus, changing all of these file locations and file paths is going to hurt me on a spiritual level. It works just fine where it is, and it makes logical sense to boot.